install.bat

```diff
@@ -471,6 +471,35 @@ local function get_visual_studio_directory()
 	return nil
 end
 
+local function get_visual_studio_directory_from_vswhere()
+	assert(type(vars.LUA_RUNTIME)=="string", "requires vars.LUA_RUNTIME to be set before calling this function.")
+	local major, minor = vars.LUA_RUNTIME:match('VCRUNTIME(%d+)(%d)$')
+	if not major or tonumber(major) < 14 then
+		print(S[[    Cannot auto-detect Visual Studio 2017 or higher from $LUA_RUNTIME]])
+		return nil
+	end
+	local vswhere_dir = os.getenv('PROGRAMFILES(X86)').."\\Microsoft Visual Studio\\Installer\\vswhere.exe"

```

Comment:
It's not a `dir`, but the path to an executable, could you rename this variable?

Reply:
Renamed, thanks!